tree/treeplayer/src/TTreeProcessorMT.cxx

```diff
@@ -348,7 +347,7 @@ void TTreeView::MakeChain(const std::vector<std::string> &treeNames, const std::
       const auto &thisFriendEntries = friendEntries[i];
 
       // Build a friend chain
-      auto frChain = std::make_unique<TChain>(thisFriendName.c_str(), "", TChain::kWithoutGlobalRegistration);
```

Comment:
just to make sure that I understand what is going on. This was thread-safety already before, right?

Reply:
yes, and the new code does exactly the same thing